src/parser/parsers/generics.py

```diff
@@ -0,0 +1,132 @@
+import utils.json_utils as json_utils
+import os
+
+
+class GenericParser:
+    """
+    Lightly parse the generic data to have more readable keys.
+
+    Validate structures used by the frontend to identify
+    possible failure points in the case they are modified
+    by Valve.
+    """
+
+    def __init__(self, output_dir, generic_data):
+        self.OUTPUT_DIR = output_dir
```

Comment:
name this generic_data_dir as output_dir at the moment is widely used to refer to the environment var

Reply:
good point